google-cloud-tools-plugin/build.gradle

```diff
@@ -31,12 +31,25 @@ intellij {
 
 project.afterEvaluate {
     prepareSandbox.dependsOn ':google-account-plugin:prepareSandbox'
+
+    //todo: currently we need this to include 'ultimate' module to the plugin zip; it would be better to support such inclusion in 'intellij' plugin
+    def task = project.tasks.findByName("prepareSandbox")
+    task.dependsOn project('ultimate').jar
+    task.doLast {
+        copy {
+            from files(project('ultimate').jar)
```

Comment:
I don't have a preference, just do what works for you, I was just confused. Didn't know files could resolve a task into an archive like that.